etl/steps/data/garden/dummy/2020-01-01/dummy.py

```diff
@@ -52,38 +48,7 @@ def run(dest_dir: str) -> None:
     # Add table of processed data to the new dataset.
     ds_garden.add(tb_garden)
 
-    # Update dataset and table metadata using the adjacent yaml file.
```

Comment:
Why it got removed? Metadata yaml file is still there.

Reply:
Thanks, that was a mistake, I just put those lines back.